pkg/cmd/dev/util.go

```diff
@@ -105,11 +107,74 @@ func addCommonTestFlags(cmd *cobra.Command) {
 	cmd.Flags().Duration(timeoutFlag, 0*time.Minute, "timeout for test")
 }
 
-func ensureBinaryInPath(bin string) error {
+func ensureBinaryInPath(dev *dev, bin string) error {
```

Comment:
Nit: Consider making this a method on `*dev` instead of passing it as a parameter

Reply:
Done